frontend/src/metabase/ui/components/typography/Title/Title.styled.tsx

```diff
@@ -0,0 +1,19 @@
+import type { MantineThemeOverride, TitleStylesParams } from "@mantine/core";
+
+export const getTitleOverrides = (): MantineThemeOverride["components"] => ({
+  Title: {
+    styles: (theme, params: TitleStylesParams) => {
+      if (params.element === "h3") {
+        return {
+          root: {
+            fontWeight: "normal",
+            letterSpacing: "0.7px",
```

Comment:
:x: `letter-spacing` breaks Arabic: https://www.w3.org/TR/alreq/#h_styling_individual_letters

We [do support Arabic](https://www.metabase.com/docs/latest/configuring-metabase/localization).

Reply:
@kamilmielnik Initially I tried using `:dir` selector to target `ltr` languages only, but it doesn't work in many (most) cases and it can still break something if there is a ltr language with connected chars. So I decided to use `:lang` and target only `en` for now, which seems safe. Wdyt?

Reply:
I've also checked that `uppercase` transform doesn't affect arabic and hebrew, and I guess any other languages where there are differences between start/middle/end characters

Reply:
> So I decided to use `:lang` and target only `en` for now, which seems safe. Wdyt?

Agreed, it's safe :+1: 
Ideally we should list all the languages that we support where we can use `letter-spacing` though.

> and target only `en` for now,

It's a good start :+1: 
Let's leave a comment for this line maybe, it's definitely not obvious why it's there.

Reply:
@kamilmielnik added a comment with a link